src/dock.cpp

```diff
@@ -56,6 +56,8 @@ Dock::Dock(const QVariantMap &config, EntitiesInterface *entities, Notifications
         }
     }
 
+    m_url = QString("ws://").append(m_ip).append(":946");
```

Comment:
See comment in setup-schema.json about `ip` naming. `hostname` would be a better name since `ip` implies that only ip addresses can be used.